src/app/components/home-page/home-page.component.scss

```
@@ -1,13 +1,31 @@
 @import "../../../styles/variables.scss";
 
 h1 {
-  font-size: xx-large;
+  font-size: 10vw;
+  line-height: 100%;
+  margin-top: min(5%, 100px);
+}
+
+h2 {
+  color: $light-purple;
+  font-size: 170%;
+  font-weight: 300;
+  margin:15px;
+}
+
+.title {
+  margin-bottom: 6vw;
+
+}
```

L'espace entre le sous-titre et la barre de séparation est trop grande pour de grands écrans.
```suggestion
.title {
  margin-bottom: 4vw;
}
```